apps/client-server/src/app/post-parsers/models/description-node/description-node-tree.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 export type InsertionOptions = {
   insertTitle?: string;
-  insertTags?: string[];
+  insertTags?: string;
```

Comment:
Tags are now passed as already stringified because stringify function is at the websiteOptions level

Reply:
See comment lower. I have opinions as this change has bloated the PR.